### PR TITLE
Updates to landing after last lesson

### PR DESCRIPTION
**Pull request by nastiaymlnv**

### CODE REVIEW

**Comment by vladhybalo** [comment]
image
I would like to see this bottom animated block visible only within image. So image`s block overflow should be hidden

**Comment by vladhybalo** [comment]
image
Looks like this blog has some hardcoded width and overflows a little bit

File: lesson6/css/media.css [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+    #hamburger-menu__toggle:checked ~ .hamburger-menu__btn .hamburger-menu__space {
+        transform: rotate(45deg);
+    }
+
+    #hamburger-menu__toggle:checked ~ .hamburger-menu__btn .hamburger-menu__space::before {
+        top: 0;
+        transform: rotate(0);
+    }
+
+    #hamburger-menu__toggle:checked ~ .hamburger-menu__btn .hamburger-menu__space::after {
```
Comment (vladhybalo) [inline lesson6/css/media.css:194]: As I remember BEM implies using class selectors instead of id

**Comment by andriiivaniuk** [comment]
I think that according to design, both vertical and horizontal gaps should be of same size

**Comment by andriiivaniuk** [comment]
It looks like the pictures are anchored to the bottom left corner, so the content is not centered on small screens. Maybe try using background-position: center to fix this?

**Comment by andriiivaniuk** [comment]
If I remember correctly, we were asked to make this underscore animating on hover.  (and all other section headers too)

File: lesson6/css/style.css [hunk 2]
```diff
@@ -1,38 +1,109 @@
...
+    --font-size-standart: 16px;
+    --font-size-m: 18px;
+    --font-size-l: 24px;
+    --font-size-xl: 30px;
+    --font-size-xxl: 35px;
+    --font-size-3xl: 50px;
+}
+
 html {
     font-family: Lato, Roboto, sans-serif;
```
Comment (andriiivaniuk) [inline lesson6/css/style.css:18]: Can you also add smooth scrolling for anchor links please?

**Comment by andriiivaniuk** [comment]
I feel like this section has little to none responsiveness. Can you add some hover effects and animations so my brain would release dopamine interacting with it? :D

**Comment by andriiivaniuk** [comment]
The video looks too narrow on 4k screen. I think it has to be the same size as main content and hold the right aspect ratio.

**Review by andriiivaniuk — APPROVED** [review]

**Review by VladyslavHybalo — APPROVED** [review]

**Pull request merged**